config/database.yml

```diff
@@ -1,92 +1,92 @@
-# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
- ### This inline comment is intended for readme file, I am writing it here since it was not under changed files
- [ ] In your install section please mention to the user that he needs to run `bundle install` and mention that he needs to create and migrate databases, write down the commands
- [ ] Please fill these 
![image](https://user-images.githubusercontent.com/62705947/215508426-07ae4b92-b9ef-4d78-b7bb-91af31d805a1.png)